storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -231,7 +183,15 @@
 
     @VisibleForTesting
     public static final List<ACL> ZK_ACLS = Arrays.asList(ZooDefs.Ids.CREATOR_ALL_ACL.get(0),
-            new ACL(ZooDefs.Perms.READ | ZooDefs.Perms.CREATE, ZooDefs.Ids.ANYONE_ID_UNSAFE));
+            new ACL(ZooDefs.Perms.READ ^ ZooDefs.Perms.CREATE, ZooDefs.Ids.ANYONE_ID_UNSAFE));
```

Comment:
Could you explain why you changed OR to XOR?

Reply:
i copy it from old nimbus.clj, i didn't get the point why 2.0 change it to or

Reply:
Old nimbus.clj uses `bit-or` not `bit-xor`.

https://github.com/apache/storm/blob/1.x-branch/storm-core/src/clj/org/apache/storm/daemon/nimbus.clj#L147
https://github.com/apache/storm/blob/1.1.x-branch/storm-core/src/clj/org/apache/storm/daemon/nimbus.clj#L147

Reply:
yeah, you are right